ui/lobby/src/ctrl.ts

```diff
@@ -215,6 +217,7 @@ export default class LobbyController {
         xhr.seeks().then(this.setSeeks);
         break;
     }
+    this.reload();
```

Comment:
why not just `playstrategy.reload();` here?

Reply:
yes that would also work, the other modules that are similar (swiss, tournament, round) have a reload function and I feel like it may be expanded in the future...